pkg/splunk/enterprise/util.go

```diff
@@ -224,11 +223,10 @@ func ApplySmartstoreConfigMap(client splcommon.ControllerClient, cr splcommon.Me
 	return SplunkOperatorAppConfigMap, configMapDataChanged, nil
 }
 
-//  setupInitContainer modifies the podTemplateSpec object to incorporate support for DFS.
-func setupInitContainer(podTemplateSpec *corev1.PodTemplateSpec, sparkImage string, imagePullPolicy string, commandOnContainer string) {
-	// create an init container in the pod, which is just used to populate the jdk and spark mount directories
+//  setupInitContainer modifies the podTemplateSpec object
+func setupInitContainer(podTemplateSpec *corev1.PodTemplateSpec, Image string, imagePullPolicy string, commandOnContainer string) {
 	containerSpec := corev1.Container{
-		Image:           sparkImage,
```

Comment:
Had an offline chat with @mgaldino-splunk about this, understand the InitContainer is used by SmartStore code. cc: @sgontla , @pdhanoyasplunk